mobile/lib/features/trade/channel_creation_flow/channel_configuration_screen.dart

```diff
@@ -0,0 +1,512 @@
+    final submitOrderChangeNotifier = context.read<SubmitOrderChangeNotifier>();
+
+    Color confirmationSliderColor =
+        widget.tradeValues.direction == Direction.long ? tradeTheme.buy : tradeTheme.sell;
+
+    Amount orderMatchingFee =
+        tradeValueChangeNotifier.orderMatchingFee(widget.tradeValues.direction) ?? Amount.zero();
+
+    Amount totalFee = orderMatchingFee.add(fundingTxFee).add(channelFeeReserve);
+    Amount totalAmountToBeFunded = ownTotalCollateral.add(totalFee);
+
+    return Scaffold(
+      body: SafeArea(
+        child: Form(
+          key: _formKey,
+          child: Container(
+            padding: const EdgeInsets.only(top: 20, left: 15, right: 15),
+            child: Column(
+              children: [
+                Column(
+                  children: [
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        Expanded(
+                          child: Stack(
+                            children: [
+                              GestureDetector(
+                                child: Container(
+                                    alignment: AlignmentDirectional.topStart,
+                                    decoration: BoxDecoration(
+                                        color: Colors.transparent,
+                                        borderRadius: BorderRadius.circular(10)),
+                                    width: 70,
+                                    child: const Icon(
+                                      Icons.arrow_back_ios_new_rounded,
+                                      size: 22,
+                                    )),
+                                onTap: () {
+                                  GoRouter.of(context).go(TradeScreen.route);
+                                },
+                              ),
+                              const Row(
+                                mainAxisAlignment: MainAxisAlignment.center,
+                                children: [
+                                  Text(
+                                    "Fund Channel",
+                                    style: TextStyle(fontWeight: FontWeight.w500, fontSize: 20),
+                                  ),
+                                ],
+                              ),
+                            ],
+                          ),
+                        ),
+                      ],
+                    ),
+                    const SizedBox(height: 10),
+                    Center(
+                      child: Text.rich(
+                        textAlign: TextAlign.justify,
+                        TextSpan(
+                          children: [
+                            TextSpan(
+                              text:
+                                  "🔔 This is your first trade and you do not have a channel yet. "
```

Comment:
In the video this is very hard to read. I also think its too much text, maybe the title is enough?

Reply:
Feel free to propose something else :)

Reply:
> maybe the title is enough?

😅